classes/api/entries.php

```diff
@@ -102,6 +118,50 @@ public function delete_item( WP_REST_Request $request ) {
 		}
 	}
 
+    /**
+     * Delete all entries of a form based on Form ID
+     *
+     * GET /cf-api/v2/entries/form-id/delete
+     *
+     * @since 1.7.0
+     *
+     * @param WP_REST_Request $request
+     *
+     * @return Caldera_Forms_API_Error|Caldera_Forms_API_Response
+     */
+    public function delete_entries( WP_REST_Request $request ) {
+        try{
+            $this->form_object_factory( $request[ 'form_id' ], $request );
+        }catch ( Exception $e ){
+            return Caldera_Forms_API_Response_Factory::error_form_not_found();
+        }
+
+        $entries = \calderawp\CalderaFormsQueries\CalderaFormsQueries()->selectByFormId( $request[ 'form_id' ], false );
+
+        if( null != $entries ) {
+            $entryIds = [];
+            foreach( array_column( $entries, 'entry' ) as $entry ){
+                $entryIds[] = $entry->id;
+            }
+
+            Caldera_Forms_Entry_Bulk::delete_entries( $entryIds );
+            $data = array(
```

Comment:
I'm not sure if this should be 200 or 204 https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete